specs-go/v1/manifest.go

```diff
@@ -27,6 +27,6 @@ type Manifest struct {
 	// Layers is an indexed list of layers referenced by the manifest.
 	Layers []Descriptor `json:"layers"`
 
-	// Annotations contains arbitrary metadata for the manifest list.
+	// Annotations contains arbitrary metadata for the manifest.
```

Comment:
This is also [in flight][1] with #484. I guess whichever way gets the fix in first… ;)

[1]: https://github.com/opencontainers/image-spec/pull/484/files#diff-83ea0fb4782f9fe8fcc3bb3063ef1b53L30